src/styles/card.scss

```diff
@@ -0,0 +1,103 @@
+$color-accent-blue: #00acdc;
+$color-main-accent: #060b35;
+$color-secondary: #616070;
+
+.card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+  width: 200px;
+  margin-bottom: 48px;
+  margin-right: 50px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    padding: 32px 19px 40px;
+  }
+
+  &__header {
+    padding: 0 16px;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    font-style: normal;
+    color: $color-main-accent;
+  }
+
+  &__paragraph {
+    padding: 4px 16px 0;
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    font-style: normal;
```

Comment:
redudndant due to this values being the default

Reply:
i'm fix it, please check my work))